FreeAgent.postman_collection.json

```diff
@@ -6385,23 +6349,64 @@
 		{
 			"listen": "prerequest",
 			"script": {
-				"id": "a760280f-3c96-412c-a1df-84982cbffe42",
 				"type": "text/javascript",
 				"exec": [
-					""
+					"// Refresh the Access Token if it expired",
```

Comment:
We probably shouldn't be including the refresh script in the collection file as it requires some extra setup -- we can just leave a `""` here.